dev/bench/bench.sh

```diff
@@ -331,7 +338,12 @@ create_opam() {
     # Rest of default switches
     opam repo add -q --set-default iris-dev "https://gitlab.mpi-sws.org/FP/opam-dev.git"
 
-    opam switch create -qy -j$number_of_processors "$OPAM_COMP"
+    if [[ $USE_FLAMBDA ]];
+    then flambda=--packages=ocaml-variants.${OCAML_VER}+options,ocaml-option-flambda
+    else flambda=
+    fi
+
+    opam switch create -qy -j$number_of_processors "$OPAM_COMP" $flambda
```

Comment:
Setting the name of the switch to `ocaml-base-compiler.$OCAML_VER` even when flambda is enabled is IMHO misleading.

Reply:
I've never understood switch create so I can't change it

Reply:
What is it that you don't understand? Basically you do `opam switch create $name $compiler` , if `$compiler` is absent (usually a really bad idea, but still working due to legacy from opam 1), opam will try to guess which compiler to use from `$name`.

Reply:
changed